spec/models/authentication/authenticated_user_spec.rb

```diff
@@ -4,16 +4,16 @@
 
 require 'support/factory_bot'
 
-RSpec.describe AuthenticatedUser do
-  subject(:authenticated_user) { described_class.new(credentials) }
+RSpec.describe Authentication::AuthenticatedUser do
+  subject(:authenticated_user) do
+    described_class.new(credentials).tap(&:validate)
+  end
 
   describe 'validations' do
```

Comment:
You've added new validation in this PR. Add new model test for validation, or validation test, or both (the best solution)

Reply:
There is a test for the validator